Lib/fontParts/base/bPoint.py

```diff
@@ -21,16 +21,16 @@ def _setPoint(self, point):
 
     def __eq__(self, other):
         if hasattr(other, "_point"):
-            return self._point == other._point
-        return NotImplemented
+            return id(self._point) == id(other._point)
+        return False
```

Comment:
You were correct indeed. You can return NotImplemented here. Python takes care of returning False.